schema/rdl_schema_0.1.json

```
@@ -48,20 +50,24 @@
 			"format": "date"
 		},
 		"version": {
+			"title": "Dataset version",
 			"type": "integer",
-			"description": "Version of the dataset"
+			"description": "The version of the dataset."
 		},
 		"purpose": {
+			"title": "Dataset purpose",
 			"type": "string",
-			"description": "Purpose for what the data has been produced"
+			"description": "The purpose the dataset was produced for."
 		},
 		"project": {
+			"title": "Project title",
 			"type": "string",
-			"description": "Project under which data has been produced"
+			"description": "The name of the project that the dataset was created as part of."
```

Copy edit for clarity and brevity:

```suggestion
			"description": "The name of the project that produced the dataset."
```